torch_xla/csrc/reduction.cpp

```diff
@@ -74,8 +74,9 @@ xla::XlaOp CreateSummation(
       dimensions);
   if (scale) {
     xla::XlaOp scale = XlaHelpers::ScalarValue<float>(
-        1.0f / static_cast<float>(rinfo.element_count), shape.element_type(),
-        input.builder());
+        rinfo.element_count > 0 ? 1.0f / static_cast<float>(rinfo.element_count)
+                                : NAN,
```

Comment:
I'd `#include <cmath>` and run `clang-format`.

Reply:
Hmmm `clang-format` seems to be happy with this line ;) 
Btw somehow it builds and passes test without cmath...hmmm but I just added anyway

Reply:
Oh, I know it compiles. But relies on some other include to pull the proper include, which is wrong.
If that include will stop including cmath, we will be suddenly breaking.